cython_shadow/cython.py

```diff
@@ -0,0 +1 @@
+../cython.py
```

Comment:
I think this file should not be symbolic link but new file since this file imports also `Cython.Compiler.Main`

Reply:
For now I think I'll catch any import error and print a message.